src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -397,6 +399,82 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructorReminders(
                 EmailType.FEEDBACK_UNPUBLISHED.getSubject());
     }
 
+    public EmailWrapper generateFeedbackSessionResendLinksEmail(String userEmail) {
+        Instant endTime = new Date().toInstant();
+        Calendar calendar = Calendar.getInstance();
+        calendar.add(Calendar.MONTH, -6);
+        Instant startTime = calendar.getTime().toInstant();
+        String subject = EmailType.FEEDBACK_ACCESS_LINKS_RESENT.getSubject();
+        StringBuffer linksFragmentValue = new StringBuffer(1000);
+        String emailBody;
+        String studentName = "";
```

Comment:
This is because the form that the student submitted only contains the email address. We can only know the student's name if the student currently has a feedback session. However, I need to initialise the student attribute here as I am adding it to an email at the later part.

Reply:
I mean what happened if `sutdentName` is initialized to just a null string just like `emailBody` above?

Reply:
@LuLechuan how about this?

Reply:
Sorry forgot about this!

Reply:
@bqnguyen94 I got what you meant. But the `studentName` and `emailBody` are different cases. `emailBody` will be initialised no matter what, but `studentName` might not have a value. This means if I set it a null string, it will cause an error later coz it might not be initialised.

Reply:
In what case the null `studentName` will cause errors?

Reply:
If I initialise it as a null value, then I cannot even call the condition `if (studentName == null)` to check whether the `studentName` has already assigned the value as `studentName` is not initialised (line 418). Even if I remove the checking condition and assign `studentName` every time I found a related feedback session, I will still get an error at line 468 as the variable is not initialised.

Reply:
I see, ok.